manifests/init.pp

```diff
@@ -220,6 +239,11 @@
   Stdlib::Absolutepath $rdeck_home                    = $rundeck::params::rdeck_home,
   Optional[String] $rdeck_profile_template            = undef,
   String $realm_template                              = $rundeck::params::realm_template,
+  Stdlib::HTTPUrl $repo_yum_source                    = $rundeck::params::repo_yum_source,
+  String $repo_yum_gpgkey                             = $rundeck::params::repo_yum_gpgkey,
+  Stdlib::HTTPUrl $repo_apt_source                    = $rundeck::params::repo_apt_source,
+  String $repo_apt_key_id                             = $rundeck::params::repo_apt_key_id,
+  String $repo_apt_keyserver                          = $rundeck::params::repo_apt_keyserver,
```

Comment:
not sure if there's a pattern for hostnames?

Reply:
I don't think so